### ISSUE TITLE
Vertical overview doesn't show unless Super key pressed.

### ISSUE THREAD

**Issue by horga83** [RensAlthuis/vertical-overview#40]
Would be nice to have the overview visible all the time like Workspaces to Dock.

**Comment by ndim** [comment]
Please make that optional, though.

**Comment by duartec3000** [comment]
I strongly disagree. If there is the need of having the Favourites Dock and the Workspace Dock always visible there are already 2 extensions that cover that: Dash-to-Dock and Workspaces-to-Dock.

**Comment by horga83** [comment]
Well that would be great except you fail to realize that both those extensions don't work with Gnome 40

**Comment by horga83** [comment]
Any thoughts on this RensAlthuis?
